file_zbc.c

```diff
@@ -1895,7 +1895,7 @@ static int zbc_check_rdwr(struct tcmu_device *dev, struct tcmulib_cmd *cmd)
 	}
 
 	if (lba + nr_lbas > zdev->capacity || lba + nr_lbas < lba) {
-		tcmu_dev_err(dev, "cmd exceeds last lba %llu (lba %llu, xfer len %lu)\n",
+		tcmu_dev_err(dev, "cmd exceeds last lba %llu (lba %lu, xfer len %lu)\n",
```

Comment:
For these 64 bit types on 32 bit userspace doesn't it need to be long long and not just long?

Should we be doing PRId64/PRIu64 so we do not have to worry?